src/modules/forms/plugins/Cors/CorsPlugin.js

```diff
@@ -14,80 +14,101 @@ import MultiSelect from '../../../selects/MultiSelect/MultiSelect';
 import TagSelect from '../../../selects/TagSelect/TagSelect';
 
 const propTypes = {
+    apiSchema: PropTypes.object.isRequired,
     className: PropTypes.string,
+    edit: PropTypes.bool,
     name: PropTypes.string.isRequired,
     plugin: PropTypes.object.isRequired,
     pluginName: PropTypes.string.isRequired,
     handlePluginExclude: PropTypes.func.isRequired,
 };
 
-const CorsPlugin = ({ className, name, handlePluginExclude, plugin, pluginName }) => {
-    const b = block(className);
-    const optionsTransformer = config => {
-        return config.map(item => ({
-            label: item,
-            value: item,
-        }));
-    };
+class CorsPlugin extends PureComponent {
+    componentDidMount = () => {
+        this.setState({});
+    }
 
-    return (
-        <div className={b('section')()}>
-            <Row fullwidth>
-                <Row col>
-                    <Label>Plugin Name</Label>
-                    <Input input={{value: 'CORS'}} disabled />
-                </Row>
-                <ControlBar name={`${name}.enabled`} removePlugin={() => handlePluginExclude(pluginName)} />
-            </Row>
-            <Row className={b('row')()} fullwidth>
-                <Row col>
-                    <Label>Domains</Label>
-                    <Field
-                        name={`${name}.config.domains`}
-                        type="text"
-                        placeholder={SETUP.placeholders.cors.domains}
-                        component={Input}
-                    />
-                    <Hint>A list of all domains from which the endpoint will accept requests</Hint>
-                </Row>
-                <Row col>
-                    <Label>Methods</Label>
-                    <Field
-                        name={`${name}.config.methods`}
-                        type="text"
-                        placeholder={SETUP.placeholders.cors.methods}
-                        options={optionsTransformer(plugin.config.methods)}
-                        component={MultiSelect}
-                    />
-                    <Hint>HTTP methods that are supported for the endpoint.</Hint>
+    render() {
+        const { apiSchema, className, edit, name, handlePluginExclude, plugin, pluginName } = this.props;
+        const b = block(className);
+        const optionsTransformer = config => {
+            return config.map(item => ({
+                label: item,
+                value: item,
+            }));
+        };
+        const getValues = key => {
+            return plugin.config[key];
+        };
+        const allValues = key => {
+            return apiSchema.plugins[0].config[key];
+        };
+
+        return (
+            <div className={b('section')()}>
```

Comment:
if no parameters is given, perhaps it's better to call `b` function on his initialisation
```js
const b = block(className)()
b('section')
b('row')
```

Reply:
I was wrong, the way its is is right 👍